spec/forms/csv_upload_form_spec.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/file_upload_helpers'
+
+RSpec.describe CsvUploadForm, type: :form do
+  describe '#save' do
+    context 'given a VALID CSV file' do
+      it 'returns true' do
+        form = described_class.new
+        file = FileUploadHelpers.upload_file fixture: 'valid_7_keywords.csv'
+
+        success = form.save file
+
+        expect(success).to be(true)
+      end
+
+      it 'save all keywords' do
```

Comment:
```suggestion
      it 'saves all keywords' do
```

Reply:
Fixed in 73e3ef124168274de87342bde97fadb897dd54db